.github/workflows/tests.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
       - name: Install cargo-nextest
         uses: taiki-e/install-action@v2
         with:
-          tool: nextest@0.9.80
```

Comment:
We've [intentionally pinned an older version of nextest](https://github.com/n0-computer/iroh/pull/3088) due to buggy behavior.